WorkbenchProject/src/main/java/edu/uwb/braingrid/workbench/model/SimulationSpecification.java

```diff
@@ -128,30 +128,30 @@ public String getSimExecutable() {
     }
 
     /**
-     * Provides the location of the folder where the simulator is executed, where its code is
-     * downloaded to, and where it is built.
+     * Provides the location of the folder where the simulator is downloaded, built, and executed.
+     * If the simulator execution is specified to take place on a remote machine, this is relative
+     * to the root directory for the user.
```

Comment:
For clarification, I think you mean "home directory" for the user. There is only one root: "/".

Reply:
Good catch. I copied that description from the existing @return tag and didn't notice the error. I corrected this in both places in my latest commit.